README.md

```diff
@@ -7,17 +7,17 @@ Website: [phpixie.com](https://phpixie.com/)
 
 PHPixie started as a micro framework and has gradually grown to be one of the 
 most popular fullstack PHP frameworks while retaining its high performance. 
-This is in part because of the strict architecture that avoids common pitfalls 
-such as reliance on static methods, global scope, singletons and other antipatterns, 
-thus also ensuring that the code is easy to read, debug, extend and test. 
-In fact, all PHPixie components boast full unit test coverage. This PHP framework
-never stands in your way and provides you with full control over execution 
-flow. It's easy to learn and straightforward to master.
-
-But it's not all just about the code. The PHPixie community is very friendly and 
-helpful—you can expect an answer to your question within minutes of asking 
-it in the chat. The framework documentation is full of examples and is regularly 
-updated with tutorial videos.
+The reason PHPixie will be one of the most popular full stack PHP frameworks
+is that the strict architecture that avoids common pitfalls such as reliance on 
+static methods, global scope, singletons and other antipatterns, and ensures that
+the code is easy to read, debug, extend and test. In fact, all PHPixie components 
+boast full unit test coverage. PHPixie never stands in your way and provides you with
+full control over execution flow; therefore, PHPixie is easy to learn and straightforward 
```

Comment:
Changing this period to "; therefore," completely changes the meaning here. It links the two thoughts together in a way that most likely was not intended originally. Also, it's redundant to say "PHPixie" twice in a row.